indico/modules/events/editing/client/js/editing/timeline/index.jsx

```diff
@@ -46,10 +46,6 @@ export default function Timeline() {
     };
   }, [dispatch]);
 
-  const refresh = () => {
-    dispatch(actions.useUpdatedTimeline());
-  };
```

Comment:
Just out of curiosity, what was the reason for inlining this? Don't you find it more readable to have short functions in the JSX and the actual logic above?